### ISSUE TITLE
[Bug] Keyboard controls doesn't responding well [MacOS ?]

### ISSUE THREAD

**Issue by UsRnM44** [Ryujinx/Ryujinx#5849]
### Description of the issue

Starting from previous week, while playing Super Mario Wonder, I tackled the fact that the characters are moving weird and not responding well and fast. Thinking this might just being early bug of the game, I played several other games on Ryujinx, and the same problem occurring while controlling the character, I need to press really hard on the keyboard to actually make it move, my fingers actually hurts.
The situation isn't due my keyboard, playing on other emulators (such as Cemu and Dolphin) are fine, only in Ryujinx characters doesn't responding well to the controls.

This issue only happened with the last 10 or so updates, there were some updates where I haven't played much to reassure from where this starts, so it's hard for me to track it down.

This issue may be a duplicate of https://github.com/Ryujinx/Ryujinx/issues/5828 but the problem is reported differently, as it's not just pressing two keys at the times, but even just moving can be hard sometimes to push really hard on the buttons.

(I can attach log if needed)

### Reproduction steps

Basically using keyboard as the controllers. Happens to me on Super Mario Wonder specifically, but happens on every game.

### Log file

(If needed, I can attach)

### OS

MacOS Ventura 13.6

### Ryujinx version

1.1.1065

### CPU

Intel

### GPU

AMD

### RAM

16GB

**Comment by UsRnM44** [comment]
Seems that this issue doesn't get it's attention. I updated the latest version to see if its fixed, but no, just the same.
Just to show how this is limiting movement, I recorded the next video. The left circle pad in this situation is on the arrows keys just to show. Just see the last 5 seconds how I couldn't move Mario to the right or left, but sticked only forward.

https://github.com/Ryujinx/Ryujinx/assets/122369709/d5c5b594-f0ee-4f7d-a3ce-69bf7fe6c9d7

**Comment by UsRnM44** [comment]
@piplup55 All I want is that at least the developers would know this is an issue.

**Comment by piplup55** [comment]
> @piplup55 All I want is that at least the developers would admit this is an issue.

no one said it wasn't an issue, i just said the other issue was duplicate of this one
